src/main/resources/application.properties

```
@@ -2,3 +2,4 @@ spring.datasource.url=jdbc:mysql://localhost:3306/sampledb?characterEncoding=UTF
 spring.datasource.username=root
 spring.datasource.password=hcut84112
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+mybatis.mapper-locations=classpath*:/com/example/demo/auth/UserRepository.xml
```

これだとUserRepository.xml固定になっちゃいそうなので
```suggestion
mybatis.mapper-locations=classpath*:/com/example/demo/*.xml
```